app/routes/index.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+    session: Ember.inject.service(),
+
+    model() {
+        if (!this.get('session.isAuthenticated')) return [];
+        return this.get('store').query('preprint-provider', {
+            'filter[reviews_workflow]': 'None',
```

Comment:
These filters could be written more neatly as an object, though it doesn't much matter:
```
{
  filter: {
    reviews_workflow: 'None', 
    permissions: 'set_up_moderation'
  }
}
```

It's kinda gross that the API expects `'None'` like that... but I guess it works for now.

Reply:
Neato, did not know you could do that